organisations/relationship_transfer.go

```diff
@@ -68,20 +68,31 @@ func getNodeRelationshipNames(cypherRunner neoutils.CypherRunner, uuid string) (
 	return relationshipsFromNodeWithUUID, relationshipsToNodeWithUUID, nil
 }
 
+func constructTransferRelationshipsWithPlatformVersionFromNodeQuery(fromUUID string, toUUID string, predicate string) *neoism.CypherQuery {
+	transferAnnotationsQuery := &neoism.CypherQuery{
+		Statement: fmt.Sprintf(`MATCH (oldNode:Organisation {uuid:{fromUUID}})-[oldRel:%s]->(p)
+					WHERE EXISTS(oldRel.platformVersion)
+					MATCH (newNode:Organisation {uuid:{toUUID}})
+					MERGE (newNode)-[newRel:%s{platformVersion:oldRel.platformVersion}]->(p)
+					on create SET newRel = oldRel
```

Comment:
I was wondering if there is a slightly different way to do this that doesn't need to transfer all the relationships if we are creating the new node. Not sure this will help but I was wondering if we could merge the new node into the old node and there by updating the canonical uuid and adding an upp identifier but then reducing the need to transfer the relationships. Note sure how that would benefit the case of having two existing orgs i.e. which one would be mung into the end one